src/seedu/addressbook/data/menu/Menu.java

```diff
@@ -78,7 +78,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return this.name.fullName;
+        return getAsText();
```

Comment:
Just wondering why you changed here. getAsText() returns everything while this.name.fullname returns name only right?

Reply:
The reason is that toString() is used for displaying all details of the added menu
Furthermore, having a full getAsText() for toString() does not  create conflict to different name value creates the same toString()
And also the fact that there is a dedicated method for getting the name from the menu